setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     paramiko
     ruamel.yaml
     furl
-    pydantic[email]>=1.8.2
+    pydantic[email]>1.8.1,<2.0.0
```

Comment:
Make it - 
pydantic[email]>=1.8.2,<2.0.0

This way the min. working version will be explicit.

Reply:
Fixed.